test/system/800-config.bats

```diff
@@ -249,4 +249,23 @@ EOF
     run_podman rm -f -t0 $cname
 }
 
+@test "podman push CONTAINERS_CONF" {
+    skip_if_remote "containers.Conf does not effect client side of --remote"
+
+    CONTAINERS_CONF=/dev/null run_podman push --help
+    assert "$output" =~ "--compression-format string.*compression format to use \(default \"gzip\"\)" "containers.conf should set default to gzip"
+    assert "$output" != "--compression-level int.*compression level to use \(default" "containers.conf should not set default compressionlevel"
```

Comment:
This is a NOP, it will always succeed because of `!=`. Try `!~` (bang tilde). I would also suggest stripping the LHS, making it the shortest possible: 
```
   ... !~ "compression level to use \(default" ...
```